test/lib/ex_aws/s3/upload_test.exs

```diff
@@ -22,6 +22,14 @@ defmodule ExAws.S3.UploadTest do
       assert_received :chunk_uploaded
       assert_received :completed_upload
     end
+
+    test "uploading a file with a name separated by spaces" do
+      %{status_code: status_code} =
+        ExAws.S3.put_object("sergey-test", "some file name.txt", "data")
+        |> ExAws.request!
```

Comment:
I can't accept tests like this at this time. I don't have one such bucket, and I'm very very hesitant about giving the AWS keys associated with ExAws's travis account the ability to write objects to buckets.